.travis.yml

```diff
@@ -64,17 +64,13 @@ jobs:
       deploy:
         - provider: script
           script: bash docker/docker_push
+          skip_cleanup: true
           on:
-            tags: true
-            branch: 
-              - development
-              - master
-            
+            all_branches: true
```

Comment:
Do not push docker images for all branches, but only for master (releases) or development (nightly) .